### ISSUE TITLE
Change missing ref loaders error to warning

### ISSUE THREAD

**Issue by at055612** [gchq/stroom#3231]
v7.0.20 introduced a change to the reference lookups that made them error if no ref loaders were configured on the xslt filter element.  This makes sense as without any ref loaders the lookup(s) can never return anything.  However this is causing processing to error where previously it would have been processed with unsuccessful lookups.

Plan is to change it to WARNING for v7.0 and then change back to ERROR for v7.1.

Error msg is this in AbstractLookup
```
No reference loaders are configured so the lookup cannot be performed.
```

**Comment by at055612** [comment]
Changed back to warning

![image](https://user-images.githubusercontent.com/22818309/218276916-bdf90c54-81d4-4304-b5e0-f8e00fecc30f.png)

**Comment by at055612** [comment]
Fixed in > 7.0.20
